packages/components/src/templates/next/components/native/Table/Table.tsx

```diff
@@ -102,7 +105,20 @@ const Table = ({ caption, content }: TableProps) => {
                           : ""
                       }`}
                     >
-                      <Prose content={cell.content} />
+                      {cell.content.map((cellContent, index) => {
+                        if (cellContent.type === "divider") {
+                          return <Divider key={index} {...cellContent} />
+                        } else if (cellContent.type === "orderedList") {
+                          return <OrderedList key={index} {...cellContent} />
+                        } else if (cellContent.type === "paragraph") {
+                          return <Paragraph key={index} {...cellContent} />
+                        } else if (cellContent.type === "unorderedList") {
+                          return <UnorderedList key={index} {...cellContent} />
+                        } else {
+                          const _: never = cellContent
+                          return <></>
```

Comment:
Hmm but this would be caught at build time no? Which was the addition of the type guard. I'm not sure if we could potentially encounter a case here that the type guard would not catch..